docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: "2"
```

Comment:
You can't build a Docker image from the repo itself. You need to fetch a tarball or `make bin-dist`